src/collector.js

```diff
@@ -76,6 +77,18 @@ export default class Collector {
     return false;
   }
 
+  recordScannedFile(filename, scanner) {
+    // TODO: Add some code that verifies and normalizes `filename`
+    // to better avoid duplicates.
+    if (filename in this.scannedFiles) {
+      if (!this.scannedFiles[filename].includes(scanner)) {
+        this.scannedFiles[filename].push(scanner);
```

Comment:
The primary focus and usecase of this property is to log the files that were scanned though, the array of scanners is there for debugging. We will have the need to access this list more often in the future (e.g also for new checks in autoapprove stuff heading to AMO) and always reversing it to just get a list of filenames feels wrong to me.